src/material-experimental/mdc-slide-toggle/slide-toggle.spec.ts

```diff
@@ -1 +1,786 @@
-// TODO: copy tests from existing mat-slide-toggle, update as necessary to fix.
+import {BidiModule, Direction} from '@angular/cdk/bidi';
+import {dispatchFakeEvent} from '@angular/cdk/testing';
+import {Component} from '@angular/core';
+import {
+  ComponentFixture,
+  fakeAsync,
+  flush,
+  flushMicrotasks,
+  TestBed,
+  tick,
+} from '@angular/core/testing';
+import {FormControl, FormsModule, NgModel, ReactiveFormsModule} from '@angular/forms';
+import {By} from '@angular/platform-browser';
+import {MatSlideToggle, MatSlideToggleChange, MatSlideToggleModule} from './index';
+import {MAT_SLIDE_TOGGLE_DEFAULT_OPTIONS} from './slide-toggle-config';
+
+describe('MatSlideToggle without forms', () => {
```

Comment:
* `should not trigger the click event multiple times` - this one is tied to not firing the click event multiple times when dragging, if I remember correctly. Since we removed the dragging gesture, it doesn't make sense anymore.
* `should focus on underlying input element when the host is focused` - based on our discussion with @jelbourn from a couple of weeks ago, I removed the `tabindex` from the host which means that it can't receive focus anymore.